packages/@aws-cdk/aws-cognito/test/integ.user-pool-explicit-props.ts

```diff
@@ -26,9 +26,14 @@ const userpool = new UserPool(stack, 'myuserpool', {
     email: true,
```

Comment:
A quick check - were you able to deploy a CDK stack with different values to `mutable` and verify that the user pool respects it?